lib/mongo/error.rb

```diff
@@ -120,6 +121,25 @@ def label?(label)
       @labels.include?(label)
     end
 
+    # Does the write concern error have the given label?
+    #
+    # @param [ String ] label The label to check for the presence of.
+    #
+    # @return [ Boolean ] Whether the write concern error has the given label.
+    def write_concern_error_label?(label)
+      @write_concern_error_labels.include?(label)
+    end
+
+    # Whether the error has a particular label as a standard error label or an
+    # error label associated with the write concern error.
+    #
+    # @param [ String ] label The label to check for the presence of.
+    #
+    # @return [ Boolean ] Whether the error has the given label.
+    def label_or_write_concern_label?(label)
```

Comment:
Can this method be removed and the `or` done inline in the  3 call sites please?